ncm-network/src/main/pan/components/network/core-schema.pan

```diff
@@ -97,7 +99,7 @@ type structure_ethtool_ring = {
             s  Enable SecureOn(tm) password for MagicPacket(tm)
             d  Disable (wake on nothing).  This option clears all previous option
 }
-type structure_ethtool_wol = string with match (SELF, '^p|u|m|b|a|g|s|d$');
+type structure_ethtool_wol = string with match (SELF, '^[pumbagsd]+$');
```

Comment:
This changes what passes validation, for example `pppppppp` would now pass. Is this okay?

Reply:
heh, dunno, but i'll change it back to old regex. should be replaced with `choice` type anyway at some point

Reply:
Ok